### PR TITLE
Rubocop

### LINKED ISSUE #12: Rubocop setup
Setting up rubocop

### PR DESCRIPTION
**Pull request by kamok**
Add Rubocop (linter) to project; fixes #12

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -50,47 +52,45 @@
...
+  #   # Print the 10 slowest examples and example groups at the
+  #   # end of the spec run, to help surface which specs are running
+  #   # particularly slow.
+  #   config.profile_examples = 10
+  #
+  #   # Seed global randomization in this process using the `--seed` CLI option.
+  #   # Setting this allows you to use `--seed` to deterministically reproduce
+  #   # test failures related to randomization by passing the same `--seed` value
+  #   # as the one that triggered the failure.
+  #   Kernel.srand config.seed
```
Comment (cohesivejones) [inline spec/spec_helper.rb]: Some of these comments should be uncommented

File: spec/features/landing_page_spec.rb [hunk 2]
```diff
@@ -23,55 +32,52 @@
...
-      expect(page).to have_content 'Ruby'
-      expect(page).to have_content 'High'
-      expect(page).to have_content Date.today.to_s(:mdy)
-      expect(page).to have_css('svg.closebox-done')
+    it 'displays unclaimed tests type' do
+      expect(test_row).to have_content 'Python'
     end
 
-    find('svg.closebox-done').click
+    it 'displays unclaimed tests priority' do
```
Comment (cohesivejones) [inline spec/features/landing_page_spec.rb]: feature specs can and should have multiple expectations in one `it` block.
please revert this change

File: spec/spec_helper.rb (hunk 1 above)
Comment (kamok) [reply spec/spec_helper.rb]: done

File: spec/features/landing_page_spec.rb (hunk 2 above)
Comment (kamok) [reply spec/features/landing_page_spec.rb]: reverted
